ITGlueAPI/Resources/Passwords.ps1

```diff
@@ -176,25 +155,7 @@ function Set-ITGluePasswords {
         $resource_uri = $resource_uri + ('?show_password=true') # using $True in PS results in 'True' (uppercase T), so explicitly writing out 'true' is needed
```

Comment:
`show_password` shouldn't go in $resource_uri; it should be parsed as a query string parameter and added by the internal function.

Reply:
I did some testing and confirmed that `show_password` only works in the query string, even for POST or PATCH requests.